build.gradle

```diff
@@ -186,6 +203,19 @@ war {
 	from(BUILD_OTHER_DOC_DIR) { into "/WEB-INF/classes/$IN_JAR_DOC_DIR" }
 }
 
+// This is a filthy hack to get jitpack.io to not try to build the war file.
+// Since the war depends on buildDocs which depends on sphinx, the build will fail
```

Comment:
😖 

are there other docs builders that jitpack does support or does it just not do docs?

Reply:
Jitpack is a java specific tool, so it does javadoc